src/pages/AddTeam/index.js

```diff
@@ -163,20 +174,84 @@ export default function AddTeam() {
       dispatch(addTeam({ team: newTeam }));
       setCreateTeamLoading(false);
       navigate(-1);
+      toast.success(`Team ${newTeam.name} was successfully created`, {
+        position: 'top-center'
+      })
     } catch (err) {
       console.log('ON CREATE TEAM API ERR: ', err);
```

Comment:
} catch (err) {
  toast.error('Failed to update team. Please try again.');
  setUpdateTeamLoading(false);
}